setup.py

```diff
@@ -48,8 +48,9 @@
         'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.2',
-        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.5',
```

Comment:
can you add 3.4 here for completeness sake

Reply:
Left it out on purpose, because I expect us to drop it rather soon (mid-2019, when oldstable moves to current stable).

Reply:
well, everything else says 3.4. keeping this consistent and dropping it in all places is the right way to go IMO.